### PR TITLE
Merging R2 changes to R2.1

### PR DESCRIPTION
**Pull request by ATedy**
### Change description ###

Enter a description.

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SPTRIBS-

**Before merging a pull request make sure that:**

- [ ] tests have been updated / new tests has been added (if needed)
- [ ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
     <suppress>
       <notes>jackson-databind-2.13.1.jar</notes>
       <cve>CVE-2020-36518</cve>
+      <cve>CVE-2023-35116</cve>
```
Comment (osheaf01) [inline config/owasp/suppressions.xml:19]: This should already be there, pull the latest and you won't need to add this

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Just one change needed - don't duplicate the suppression
Otherwise LGTM

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (ATedy) [reply config/owasp/suppressions.xml:19]: I have pulled the latest from R2.1 but I don't see this specific supression

**Review by ElifDenizEsea — APPROVED** [review]

File: config/owasp/suppressions.xml (hunk 1 above)
Comment (osheaf01) [reply config/owasp/suppressions.xml:19]: Sorry, it's a 2.0. change. Resolving.

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**